servers/account-data-deleter/src/queueHandlers/queueHandler.ts

```diff
@@ -0,0 +1,164 @@
+import { EventEmitter } from 'events';
+import {
+  SQSClient,
+  ReceiveMessageCommand,
+  DeleteMessageCommand,
+  Message,
+  ReceiveMessageCommandOutput,
+} from '@aws-sdk/client-sqs';
+import * as Sentry from '@sentry/node';
+import { config } from '../config';
+import { SqsMessage } from '../routes/queueDelete';
+import { setTimeout } from 'timers/promises';
+import { SeverityLevel } from '@sentry/types';
+import { unleash } from '../unleash';
+import { type Unleash } from 'unleash-client';
+import { serverLogger } from '@pocket-tools/ts-logger';
+import { QueueConfig } from '../types';
+
+export abstract class QueueHandler {
```

Comment:
contains shared logic of the two queue handlers (pulling messages, deleting messages if successful, sending event for next poll)